src/cascadia/TerminalApp/TerminalPage.xaml

```diff
@@ -91,24 +91,17 @@ the MIT License. See LICENSE in the project root for license information. -->
             </TextBlock>
         </ContentDialog>
 
-
-        <ContentDialog
-            x:Load="False"
-            x:Name="KeyboardServiceDisabledDialog"
-            x:Uid="KeyboardServiceDisabledDialog"
-            DefaultButton="Primary">
-
-            <TextBlock
-                Foreground="{ThemeResource ErrorTextBrush}"
-                IsTextSelectionEnabled="True"
-                TextWrapping="WrapWholeWords"
-                Text="{x:Bind KeyboardServiceDisabledText, Mode=OneWay}" />
-        </ContentDialog>
-
         <local:CommandPalette
             x:Name="CommandPalette"
             Grid.Row="1"
             Visibility="Collapsed"
             VerticalAlignment="Stretch" />
+
+        <mux:InfoBar x:Name="KeyboardWarningInfoBar"
+                     IsClosable="True"
+                     IsIconVisible="True"
+                     IsOpen="False"
```

Comment:
consider using `x:Load=false` to completely remove this element from the tree for 99% of our population.

Reply:
holy gosh I didn't know that existed